### ISSUE TITLE
Improvement: import/export saves per game

### ISSUE THREAD

**Issue by ruvaldak** [philj56/gbcc-android#60]
We can export all GBCC saves, but we cannot export saves for singular games, and can't import saves at all. Becomes an issue if, say, you want to move a save from one emulator to another, or if you get a new device and want to move your save from your old one to the new one. Without imports, you're forced to start a new save from scratch, even though you can export the saves.

**Comment by philj56** [comment]
You can actually import saves at the moment, but it's not very obvious. The import button also lets you select `.sav`s, and they will be imported, and should work fine long as they're named the same as the corresponding game.

But yeah, the file management in general is a little obscure - I wrote it at a time when Google was making [SAF](https://developer.android.com/guide/topics/providers/document-provider) mandatory, and it was unclear whether it was going to be possible to do normal file management stuff in the future. I plan to improve it at some point.

**Comment by ruvaldak** [comment]
> You can actually import saves at the moment, but it's not very obvious. The import button also lets you select `.sav`s, and they will be imported, and should work fine long as they're named the same as the corresponding game.
> 
> But yeah, the file management in general is a little obscure - I wrote it at a time when Google was making [SAF](https://developer.android.com/guide/topics/providers/document-provider) mandatory, and it was unclear whether it was going to be possible to do normal file management stuff in the future. I plan to improve it at some point.

Oh wow, that is obscure. There was no hint or anything indicating we could import zips or savs at all.

**Comment by philj56** [comment]
Yeah, as well as file management stuff, one of the things on my [todo list](https://github.com/philj56/gbcc-android/issues/1) is to make a guide / faq in the app, to make it much more clear that you can do things like that.
